File: utils_store/src/main/java/com/hp/utils_store/utils/TypefaceUtil.java
package com.hp.utils_store.utils;

import android.content.Context;
import android.graphics.Typeface;

/**
 * 字体工具类
 */
public class TypefaceUtil {

    private static Typeface condensed;

    /**
     * 在Application初始化
     */
    public static void init(Context context){
        condensed = Typeface.createFromAsset(context.getAssets(), "fonts/DIN Condensed Bold.ttf");
    }

    /**
     * 通过{TextView.setTypeface()}设置
     */
    public static Typeface getCondensed(){
        return condensed;
    }

}

File: utils_store/src/main/java/com/hp/utils_store/utils/TypefaceUtil.kt
package com.hp.utils_store.utils

import android.content.Context
import android.graphics.Typeface
import com.hp.utils_store.utils.TypefaceUtil

/**
 * 字体工具类
 */
object TypefaceUtil {
    /**
     * 通过{TextView.setTypeface()}设置
     */
    var condensed: Typeface? = null
        private set

    /**
     * 在Application初始化
     */
    fun init(context: Context) {
        condensed = Typeface.createFromAsset(context.assets, "fonts/DIN Condensed Bold.ttf")
    }
}